### PR TITLE
ENH: Update CI / packaging for ITK 5.3.0

### PR DESCRIPTION
**Pull request by thewtex**
Explicit specification of the `imageio` is not required.

### CODE REVIEW

**Review by tbirdso — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -61,8 +61,7 @@ Usage
 
   import itk
 
-  imageio = itk.ScancoImageIO.New()
-  image = itk.imread('myvolume.ISQ', imageio=imageio)
+  image = itk.imread('myvolume.ISQ')
```
Comment (dzenanz) [inline README.rst:64]: Which change brought this improvement?

**Review by dzenanz — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (thewtex) [reply README.rst:64]: itk-5.3.0 (also present in recent RC's).

Comment (dzenanz) [reply README.rst:64]: I meant which PR in ITK, or commit SHA is responsible. Just curious.

Comment (thewtex) [reply README.rst:64]: https://github.com/InsightSoftwareConsortium/ITK/pull/3156

**Comment by thewtex** [comment]
I will tag the release and update the required Python package version for notebook CI in follow-up. I think we have an itkwidgets/itk-ioscanco dependency conflict.

**Pull request merged**